composer.json

```diff
@@ -14,9 +14,8 @@
         "issues": "https://github.com/acquia/http-hmac-php/issues"
     },
     "require": {
-        "php": ">=5.5.0",
-        "psr/http-message": "~1.0.0",
-        "sarciszewski/php-future": "~0.4.2"
+        "php": "~5.6 || ~7.0",
```

Comment:
If it's only deprecated at this point, it should still support 5.5 for now right?

Reply:
This will get merged after the 3.4.0 release and be part of the 4.0.0 release, so at that point it will no longer support it.